### PR TITLE
NMS-9941: Use the .equals() method when comparing nodeId in EventUtil…

### PR DESCRIPTION
**Pull request by UberPinguin**
…s.eventsMatch.

JIRA: https://issues.opennms.org/browse/NMS-9941

### CODE REVIEW

File: opennms-model/src/main/java/org/opennms/netmgt/model/events/EventUtils.java [hunk 1]
```diff
@@ -572,7 +572,7 @@ public static boolean eventsMatch(final Event e1, final Event e2) {
         		return false;
         }
     
-        if (e1.getNodeid() != e2.getNodeid()) {
+        if (!e1.getNodeid().equals(e2.getNodeid())) {
```
[the remark this answers is not in the document]
Comment (UberPinguin) [reply opennms-model/src/main/java/org/opennms/netmgt/model/events/EventUtils.java]: Ah, you'd get a NPE of course. I've updated the PR with an alternative solution.

**Comment by RangerRick** [comment]
Oh, heh, didn't see you already did a PR for this.  :)  I actually solved it using `Objects.equals()` to simplify the code.  https://github.com/OpenNMS/opennms/tree/jira/NMS-9941

**Comment by RangerRick** [comment]
Also it actually affects Meridian 2016 as well.

**Comment by UberPinguin** [comment]
Closing in favor of PR 1916, which is a better fix.

**Pull request closed without merging**